conans/test/functional/toolchains/cmake/test_file_api.py

```diff
@@ -26,7 +26,7 @@ def test_file_api():
     conanfile = textwrap.dedent("""
         from conans import ConanFile
         from conan.tools.cmake import CMake, CMakeFileAPI
-        from conans.model.cpp_package import CppPackage
+        from conan.tools.files import CppPackage
```

Comment:
I commented it in a review before: nothing can be imported from recipes that is not in the ``conan.tools`` namespace. So I have moved ``CppPackage`` there.

Reply:
that's not completely true, many recipes import:
```
from conans.model import Version
from conans.error import ConanException
```

Reply:
That is the old thing. Conan 2.0 will change that (maybe the only exception will be ``conan.error``. But certainly will not import from ``conans.model`` anymore. New code should not use it anymore.